ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/map-actions/map-actions.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import { readableColor } from 'polished'
+
+type Props = {
+  overlayActions: any
+  currentOverlayUrl: string
+  overlayImage: (event: any) => void
+  hasMapActions: any
+}
+
+type OverlayAction = {
+  url: string
+  description: string
+  overlayText: string
+}
+
+const Header = styled.div`
+  text-align: left;
+  font-size: ${props => props.theme.largeFontSize};
+  font-weight: bolder;
+  opacity: 0.8;
+`
+
+const Divider = styled.div`
+  height: ${props => props.theme.borderRadius};
+  margin: ${props => props.theme.minimumSpacing} 0px;
```

Comment:
❓ Why have both the minimum spacing variable and 0px here?

Reply:
This shorthand is actually
```
margin-top: ${props => props.theme.minimumSpacing}
margin-bottom: ${props => props.theme.minimumSpacing}
margin-left: 0px
margin-right: 0px
```
It is annoying that CSS makes you remember but this is definitely better.